charts/openshift-metering/templates/presto/_presto_helpers.tpl

```diff
@@ -45,6 +45,37 @@ connector.name=blackhole
 connector.name=memory
 {{ end }}
 
+{{- define "presto-prometheus-catalog-properties" -}}
+{{- if .Values.presto.spec.config.connectors.prometheus.enabled }}
+{{- with .Values.presto.spec.config.connectors.prometheus -}}
+connector.name=prometheus
+{{- if .config.uri }}
+prometheus-uri={{ .config.uri }}
+{{- else }}
+prometheus-uri=http://localhost:9090
+{{- end }}
+{{- if .config.chunkSizeDuration }}
+query-chunk-size-duration={{ .config.chunkSizeDuration }}
+{{- else }}
+query-chunk-size-duration=1h
+{{- end }}
+{{- if .config.maxQueryRangeDuration }}
+max-query-range-duration={{ .config.maxQueryRangeDuration }}
+{{- else }}
+max-query-range-duration=1d
+{{- end }}
+{{- if .config.cacheDuration }}
+cache-duration={{ .config.cacheDuration }}
+{{- else }}
+cache-duration=30s
+{{- end }}
+{{- if .auth.useServiceAccountToken }}
```

Comment:
Hmm, this seems off now too, as we can set the bearer-token-file through the `auth.useServiceAccountToken` as-well-as explicitly defining the path to the token file via `auth.bearerTokenFile`.

Should it be something like this:
```yaml
{{- if or .auth.useServiceAccountToken .auth.bearerTokenFile }}
bearer-token-file={{ .auth.bearerTokenFile }}
```

Reply:
Fixed in: 0c29d3ad